src/commands/index.js

```diff
@@ -61,7 +61,28 @@ const ignoreChannel = {
   admin: true,
 };
 
-const commands = [helloWorld, ignore, reminder, birthday, ignoreChannel];
+const roll = {
+  name: "roll",
+  action: async (message) => {
+    const args = message.content.toLowerCase().split(" ");
+    const diceNumbers = args[1].split("d").map(nb => Number(nb));
+    if (args[1] && diceNumbers[0] && diceNumbers[1]) {
```

Comment:
you should also check that diceNumbers[0] and diceNumbers[1] are actually numbers (use isNaN(Number(diceNumbers[0]) ,  for example